core/domain/user_services.py

```diff
@@ -42,6 +42,7 @@
 
+CREATOR_DASHBOARD_DISPLAY_PREFERENCES = ['card', 'list']
```

Comment:
Each of the items in this list should be named constants. Then, everywhere 'card' and 'list' are referred to, one of these constants should be used.